sample-apps/javascript-sample-app/random-metrics.js

```diff
@@ -0,0 +1,106 @@
+create_cfg = require('./config');
+
+const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector-grpc');
+const { MeterProvider } = require('@opentelemetry/metrics');
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+/** The OTLP Metrics Provider with OTLP gRPC Metric Exporter and Metrics collection Interval  */
+const meter = new MeterProvider({
```

Comment:
I think you can import the meter defined in meter.js as opposed to creating a new one, no?